src/Chapter1.hs

```diff
@@ -640,8 +658,11 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | absN `div` 10 == 0 = absN
+  | otherwise = firstDigit (absN `div` 10)
+  where absN = abs n
```

Comment:
Your implementation is correct :+1: 
The only thing you can improve is to introduce a separate helper recursive function that will perform the division. With this solution, you don't need to call `abs` on each recursive call, and it will be enough to do so only once.